packages/common/commonTypes/src/schema/index.ts

```diff
@@ -38,6 +40,8 @@ export {
   requestGroupCommonProperties,
   subject,
   taskBusinessStatus,
+  cancellationReasonCoding,
+  CancellationReasonCoding,
```

Comment:
```suggestion
  CancellationReasonCodingType,
```
To differentiate by name against the values of cancellationReasonCoding.